python/arcticdb/version_store/library.py

```diff
@@ -426,7 +428,7 @@ def _raise_if_unsupported_type_in_write_batch(payloads):
         raise ArcticUnsupportedDataTypeException(error_message)
 
     def write_batch(
-        self, payloads: List[WritePayload], prune_previous_versions: bool = True, staged=False
+        self, payloads: List[WritePayload], prune_previous_versions: bool = True, staged=False, ensure_sorted=True
```

Comment:
Needs documenting below. This will apply to every payload member?

Reply:
yes, we set it it common to all the cases, isn't it?